### DISCUSSION TITLE
How to use emoji in dataview queries?

### DISCUSSION THREAD

**Discussion by Javo-DEV** [BryanJenksCommunity/FAQ#107] [Q&A] [answered]
I would like to create a table with all files that include a specific tag e.g.  #📥️ (defined in frontmatter)
The query
table tags
from ""
where tags = "📥️" is showing no results.
If i use where tags ="Inbox", it lists the files with the tag #Inbox
So somehow I can't query for graphical tags (i.e. emojis). Is there a way to query for emojis?

**Comment by tallguyjenks** [comment, marked as answer]
This is how i do it around Dataviews limitation for emoji search in the `from` field:

![image](https://user-images.githubusercontent.com/29872822/129986774-fe89dfbc-d415-4684-91c2-a9ded943a5ea.png)

result:

![image](https://user-images.githubusercontent.com/29872822/129986799-e569f381-7b8c-412d-b079-197d639f5141.png)

**Reply by Javo-DEV** [reply]
Thanks for the fast answer, but it is somehow not working. 
table tags
from ""
where tags = "📝️/🌱️"

![image](https://user-images.githubusercontent.com/74079792/130143101-65bbcd25-2722-43ef-8032-87b1f91897cd.png)

**Reply by Javo-DEV** [reply]
![image](https://user-images.githubusercontent.com/74079792/130143487-70d5fa90-42a6-4dd3-a7c9-7aacd1c477fa.png)

**Reply by tallguyjenks** [reply]
are you copy and pasting the emoji? trying inserting them fresh into the YAML header and the dataview query
